uiplib/gui/settingsTab.py

```diff
@@ -147,6 +148,21 @@ def handle_settings(appObj):
     update_settings(appObj.new_settings)
 
 
+def update_settings_view(appObj, mainFrame):
```

Comment:
what should i name the parameter in update_settings_view , "new_settings" or "settings"??